README.rst

```diff
@@ -357,6 +353,21 @@ node.info()
     |                  |                              | - eNodeIsImportant                                            |
     +------------------+------------------------------+---------------------------------------------------------------+
 
+node.fetchValue()
+^^^^^^^^^^^^^^^^^
+
+- Returns
+
+    Promise containing fetched value when fulfilled.
+
+- Example
+
+    .. code:: javascript
+
+        client.fetchValue().then(function (value) {
```

Comment:
In C++ client this is called RequestValue(), in Java client requestValue(). We should use same terms in all our StudioAPI clients.